boilerplate/openshift/golang-osd-e2e/standard.mk

```diff
@@ -66,7 +66,7 @@ e2e-binary-build:
 
 # push e2e image tagged as latest and as repo commit hash
 .PHONY: e2e-image-build-push
-e2e-image-build-push:
+e2e-image-build-push: container-engine-login
```

Comment:
The previous version of this make target used a script and didn't login first https://github.com/openshift/boilerplate/blob/1b42d876c286bfbbfef9efcb7b01ce331f7a9afa/boilerplate/openshift/golang-osd-operator-osde2e/standard.mk#L78

Can you explain a bit why this changed and why we need to run this make target `container-engine-login` now?